src/screens/Profiles/components/ReactionBox/styles.js

```diff
@@ -9,7 +9,7 @@ export const styles = StyleSheet.create({
   },
   wrap__boxed: {
     borderRadius: 6,
-    width: cardSize.landscapeLarge.width,
+    width: Math.max(cardSize.landscapeLarge.width, 280),
```

Comment:
maybe `maxWidth: 280` in addition to line 12. But both are okay

Reply:
Woops, forgot to reset this when i made cardSizes constants.